Can't create new projects that uses prettier

### Version
3.9.2







### Environment info
```
  System:
    OS: Windows 10
    CPU: (12) x64 Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
  Binaries:
    Node: 10.16.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.9.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 42.17134.1.0
  npmGlobalPackages:
    @vue/cli: Not Found
```


### Steps to reproduce
* Run `vue create sample-app`
* Pick "Manually select features"
* Check "Linter"
* Select "Prettier"
* Pick "Lint on Save"

### What is expected?
A new project is created. 

### What is actually happening?
The following error is thrown `Error: Failed to load plugin prettier: Cannot find module 'eslint-plugin-prettier'`

<!-- generated by vue-issues. DO NOT REMOVE -->

--- a/packages/@vue/cli-plugin-eslint/generator/index.js
+++ b/packages/@vue/cli-plugin-eslint/generator/index.js
@@ -56,7 +56,9 @@ module.exports = (api, { config, lintOn = [] }, _, invoking) => {
   } else if (config === 'prettier') {
     eslintConfig.extends.push('@vue/prettier')
     Object.assign(pkg.devDependencies, {
-      '@vue/eslint-config-prettier': '^4.0.1'
+      '@vue/eslint-config-prettier': '^5.0.0',
+      'eslint-plugin-prettier': '^3.1.0',
+      prettier: '^1.18.2'
     })
     // prettier & default config do not have any style rules
     // so no need to generate an editorconfig file

